src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -689,7 +690,11 @@ def save_model(
             DeprecationWarning,
         )
         pretrained_or_pipeline = pipeline
-
+    # construct pipeline from task_name and model-id
+    if isinstance(pretrained_or_pipeline, str):
+        pretrained_or_pipeline = transformers.pipeline(
+            task=task_name, model=pretrained_or_pipeline
+        )
```

Comment:
I don't think we should load the pipeline in memory here. The whole purpose of importing API is that we don't load the model into memory

We should use `snapshot_download` instead.